.github/workflows/redpanda-build.yml

```diff
@@ -58,5 +58,5 @@ jobs:
             ccache -p # print the config
             ccache -s # print the stats before reusing
             ccache -z # zero the stats
-            ./build.sh -DCMAKE_UNITY_BUILD=ON
```

Comment:
i think this is the only change that is needed

Reply:
Done